File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -987,32 +854,28 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Sign an existing document using stream
 
-You can load the signature field from an existing PDF document and add certificate to the document using stream as follows.
+You can load the signature field from an existing PDF document by using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class and add certificate to the document using stream as follows.
```

Comment:
You can load the signature field from an existing PDF document using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class and add certificate to the document using stream as follows.

Reply:
changed the content